src/main/java/org/cbioportal/cdd/repository/topbraid/ClinicalAttributeMetadataRepositoryTopBraidImpl.java

```diff
@@ -0,0 +1,105 @@
+ * Memorial Sloan-Kettering Cancer Center
+ * has been advised of the possibility of such damage.
+*/
+
+package org.cbioportal.cdd.repository.topbraid;
+
+import java.util.*;
+
+import org.apache.log4j.Logger;
+
+import org.cbioportal.cdd.model.ClinicalAttributeMetadata;
+import org.cbioportal.cdd.repository.ClinicalAttributeMetadataRepository;
+
+import org.springframework.context.annotation.Primary;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.stereotype.Repository;
+
+/**
+ *
+ * @author Manda Wilson
+ **/
+@Primary
+@Repository
+public class ClinicalAttributeMetadataRepositoryTopBraidImpl extends TopBraidRepository<ClinicalAttributeMetadata> implements ClinicalAttributeMetadataRepository {
+
+    private final static Logger logger = Logger.getLogger(ClinicalAttributeMetadataRepositoryTopBraidImpl.class);
+
+    private String queryOverrides = "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#> " +
+        "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> " +
+        "PREFIX cdd:<http://data.mskcc.org/ontologies/clinical_data_dictionary/> " +
+        "PREFIX skos:<http://www.w3.org/2004/02/skos/core#> " +
+        "SELECT DISTINCT ?study_id ?column_header (SAMPLE(?PriorityValue) AS ?priority) (SAMPLE(?AttributeTypeValue) AS ?attribute_type) (SAMPLE(?DatatypeValue) AS ?datatype) (SAMPLE(?DescriptionValue) AS ?description) (SAMPLE(?DisplayNameValue) AS ?display_name) " +
+        "WHERE { " +
+        "    GRAPH <urn:x-evn-master:clinical_data_dictionary> { " +
+        "        ?node rdf:type ?type. " +
+        "        ?node skos:broader ?parent. " +
+        "        ?parent cdd:StudyId ?study_id. " +
+        "        ?parent skos:broader ?grandparent. " +
+        "        ?grandparent rdfs:label ?column_header. " +
+        "        OPTIONAL{?node cdd:PriorityValue ?PriorityValue}. " +
+        "        OPTIONAL{?node cdd:AttributeTypeValue ?AttributeTypeValue}. " +
+        "        OPTIONAL{?node cdd:DatatypeValue ?DatatypeValue}. " +
+        "        OPTIONAL{?node cdd:DescriptionValue ?DescriptionValue}. " +
+        "        OPTIONAL{?node cdd:DisplayNameValue ?DisplayNameValue}. " +
+        "    } " +
+        "} " +
+        "GROUP BY ?study_id ?column_header " +
+        "ORDER BY ?study_id ?column_header " +
+        "VALUES ?type {cdd:ClinicalAttributeOverridePriorityValue cdd:ClinicalAttributeOverrideAttributeTypeValue cdd:ClinicalAttributeOverrideDatatypeValue cdd:ClinicalAttributeOverrideDescriptionValue cdd:ClinicalAttributeOverrideDisplayNameValue}";
+
+    private String queryClinicalAttributes = "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#> " +
+        "PREFIX cdd:<http://data.mskcc.org/ontologies/clinical_data_dictionary/> " +
+        "PREFIX skos:<http://www.w3.org/2004/02/skos/core#> " +
+        "SELECT ?column_header ?display_name ?attribute_type ?datatype ?description ?priority " +
```

Comment:
This is the straight up query of all clinical attributes.  Since this is easier to understand, I'd recommend moving up above the override query ... so a new person reading the code would understand a little before getting to the more complicated stuff.